backend/internal/service/oidc_service.go

```diff
@@ -1211,16 +1223,12 @@ func (s *OidcService) VerifyDeviceCode(ctx context.Context, userCode string, use
 	deviceAuth.UserID = &userID
 	deviceAuth.IsAuthorized = true
 
-	if err := tx.WithContext(ctx).Save(&deviceAuth).Error; err != nil {
-		log.Printf("Error saving device auth: %v", err)
-		return err
-	}
-
-	// Verify the update was successful
```

Comment:
I can't really understand what this was for, but didn't look like it did anything, so just removed it